docker/codebook.Dockerfile

```diff
@@ -0,0 +1,27 @@
+## docker build --build-arg JHUB_VER=1.4.2 --build-arg PY_VER=3.8 --build-arg DIST=debian --build-arg DEPLOY_KEY=workflow-array-ephys-deploy.pem --build-arg REPO_OWNER=datajoint --build-arg REPO_NAME=workflow-array-ephys -f codebook.Dockerfile -t registry.vathes.com/datajoint/codebook-workflow-array-ephys:v0.0.0 .
+
+## Single Stage 
+ARG JHUB_VER
+ARG PY_VER
+ARG DIST
+FROM datajoint/djlabhub:${JHUB_VER}-py${PY_VER}-${DIST}
+
+USER root
+RUN apt-get -y update && apt-get install -y ssh git
+
+ARG DEPLOY_KEY
```

Comment:
Similar to point in other [review](https://github.com/datajoint/workflow-calcium-imaging/pull/63#pullrequestreview-926324490).